FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheker/Models/HealthStatus.cs

```diff
@@ -0,0 +1,19 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Health.Fhir.Synapse.HealthCheck.Models
+{
+    public class HealthStatus
+    {
+        public DateTimeOffset StartTime { get; set; } = DateTimeOffset.Now;
+
+        public DateTimeOffset EndTime { get; set; } = DateTimeOffset.UtcNow;
```

Comment:
I feel for health status we only need one variable like `Timestamp` to know when we get the results. The [start, end] information seems not very helpful?

Reply:
Thanks, removed endtime.